cdap-ui/app/directives/my-pipeline-configurations/my-sql-pipeline-config/my-sql-pipeline-config-ctrl.js

```diff
@@ -0,0 +1,178 @@
+/*
+* Copyright © 2017 Cask Data, Inc.
+*
+* Licensed under the Apache License, Version 2.0 (the "License"); you may not
+* use this file except in compliance with the License. You may obtain a copy of
+* the License at
+*
+* http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+* WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+* License for the specific language governing permissions and limitations under
+* the License.
+*/
+
+class MySqlPipelineConfigCtrl {
+  constructor(uuid, HydratorPlusPlusHydratorService, HYDRATOR_DEFAULT_VALUES, HydratorPlusPlusPreviewStore, HydratorPlusPlusPreviewActions, myPipelineApi, $state, myAlertOnValium) {
```

Comment:
You're not using `HYDRATOR_DEFAULT_VALUES` anywhere, are you?

Reply:
Fixed